examples/scaling-criteo/03-Training-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<p>may be rephrase <code>with only ~5 commands </code> to <code>with only ~5 lines of commands</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Lines is not correct

```
model = mm.DLRMModel(
    train.schema,
    embedding_dim=EMBEDDING_SIZE,
    bottom_block=mm.MLPBlock([128, EMBEDDING_SIZE]),
    top_block=mm.MLPBlock([128, 64, 32]),
    prediction_tasks=mm.BinaryClassificationTask(
        train.schema.select_by_tag(Tags.TARGET).column_names[0]
    )
)
```

This is more than 5 lines, but is a single command?